src/tools/tracejob.c

```diff
@@ -382,21 +382,39 @@ void
 parse_log(FILE *fp, char *job, int ind)
 {
 	struct log_entry tmp;	/* temporary log entry */
-	char buf[16384];	/* buffer to read in from file */
+	char *buf;		/* buffer to read in from file */
+	char *tbuf;		/* help wth larger entries in log */
 	char job_buf[128];	/* hold the jobid and the . */
 	char *p;		/* pointer to use for strtok */
 	int field_count;	/* which field in log entry */
 	int j = 0;
-	struct tm tms;	/* used to convert date to unix date */
+	struct tm tms;		/* used to convert date to unix date */
 	int lineno		= 0;
```

Comment:
Could you make this = 0 more consistent with the previous initialization?  Right after the variable instead of tabbed over.